### DISCUSSION TITLE
Mobile Views

### DISCUSSION THREAD

**Discussion by inLIMITit** [slabbi/uddeIM#94] [Ideas]
Hello, is there the possibility of optimizing the component for viewing on a smartphone? The bulky tables in particular often protrude beyond the screen. Seen in the view for writing a message and in general in the inbox.

**Comment by joomod** [comment]
as the uddeim component is mainly based on "old" table style it would be
a hard work to get a responsive layout.
ok, I see a layout problem on "**new message**" which I can correct by
changing the fixed **width** of the textarea (450px) to 90%.
for the In/Outbox it is harder and the best recommandation is to **use the postbox**,
which fits better

**Comment by inLIMITit** [comment]
Sounds good! :) I will try the postbox. Maybe one possibility could also be to make the table scrollable in horizontal direction?

**Comment by joomod** [comment]
I've edit the config (setup) to set the textarea in % also,
so for mobile view it is good to set the width to 90-95%
(otherwise you can keep the 0 (zero) and add this setting for the textarea to the css of the used template.)

**Comment by inLIMITit** [comment]
Thank you! I will try with the next Release :)
